<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditEnemies</class>
 <widget class="QWidget" name="EditEnemies">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>976</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Element</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_10">
      <item row="0" column="0">
       <widget class="QScrollArea" name="scrollArea_3">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>215</width>
           <height>322</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_12">
          <item row="0" column="0">
           <widget class="ABCList" name="element_widget" native="true"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="1" column="0">
      <widget class="QLineEdit" name="line_treasure">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QToolButton" name="button_change_treasure">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Treasure:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="4">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="3">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>State</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_9">
      <item row="0" column="0">
       <widget class="QScrollArea" name="scrollArea_4">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents_4">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>215</width>
           <height>322</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_13">
          <item row="0" column="0">
           <widget class="ABCList" name="state_widget" native="true"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QTableWidget" name="table_action">
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>Action</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Condition</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>turn_a</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>turn_b</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>hp</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>level</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>switch</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>kind</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>skill</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>rating</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>basic</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="1">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>DEX:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QSpinBox" name="spin_atk">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
     <item row="12" column="0" colspan="2">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Target:</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0" colspan="2">
      <widget class="QComboBox" name="combo_target"/>
     </item>
     <item row="9" column="1">
      <widget class="QSpinBox" name="spin_pdef">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>STR:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QSpinBox" name="spin_str">
       <property name="minimum">
        <number>-999</number>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="spin_dex">
       <property name="minimum">
        <number>-999</number>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QSpinBox" name="spin_eva">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>AGI:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="spin_maxsp">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>99999</number>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>MDEF:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>MaxHP:</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QSpinBox" name="spin_mdef">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QSpinBox" name="spin_agi">
       <property name="minimum">
        <number>-999</number>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>EVA:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>MaxSP:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>ATK:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QSpinBox" name="spin_int">
       <property name="minimum">
        <number>-999</number>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>PDEF:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QSpinBox" name="spin_maxhp">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>99999</number>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>INT</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="1">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Gold:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>EXP:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QSpinBox" name="spin_exp">
       <property name="maximum">
        <number>999999</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spin_gold">
       <property name="maximum">
        <number>999999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="line_name"/>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Battler</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="ImageDisplayWidget" name="label_battler_img">
          <property name="minimumSize">
           <size>
            <width>126</width>
            <height>163</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>126</width>
            <height>163</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Attacker:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combo_attacker"/>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="button_action_add">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="button_action_del">
       <property name="text">
        <string>Del</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageDisplayWidget</class>
   <extends>QLabel</extends>
   <header>dialogs/imagedisplaywidget.h</header>
  </customwidget>
  <customwidget>
   <class>ABCList</class>
   <extends>QWidget</extends>
   <header>editors/abclist.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>line_name</tabstop>
  <tabstop>spin_maxhp</tabstop>
  <tabstop>spin_maxsp</tabstop>
  <tabstop>spin_str</tabstop>
  <tabstop>spin_dex</tabstop>
  <tabstop>spin_agi</tabstop>
  <tabstop>spin_int</tabstop>
  <tabstop>spin_atk</tabstop>
  <tabstop>spin_pdef</tabstop>
  <tabstop>spin_mdef</tabstop>
  <tabstop>spin_eva</tabstop>
  <tabstop>combo_attacker</tabstop>
  <tabstop>combo_target</tabstop>
  <tabstop>spin_exp</tabstop>
  <tabstop>spin_gold</tabstop>
  <tabstop>line_treasure</tabstop>
  <tabstop>button_change_treasure</tabstop>
  <tabstop>table_action</tabstop>
  <tabstop>scrollArea_3</tabstop>
  <tabstop>scrollArea_4</tabstop>
  <tabstop>button_action_add</tabstop>
  <tabstop>button_action_del</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
